src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -464,7 +464,9 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 			}
 		}
 
-		const newHeight = templateData.rowContainer.offsetHeight;
+		// When going from welcome content to actual chat list items, rowContainer.offsetHeight is initially 0,
```

Comment:
Do you know why? Is the list being rendered too early before its visible?

Reply:
No, when `templateData.rowContainer.offsetHeight` is `0`, `this._isVisible` is already `true`, and `templateData.rowContainer.hidden` is `false`